content/developer/reference/frontend/owl_components.rst

```diff
@@ -150,13 +150,138 @@ checkboxes or datepickers. This page explains how to use these generic component
 
    * - Technical Name
      - Short Description
+   * - :ref:`ActionSwiper <frontend/owl/actionswiper>`
+     - a swiper component to perform actions on touch swipe
    * - :ref:`CheckBox <frontend/owl/checkbox>`
      - a simple checkbox component with a label next to it
    * - :ref:`Dropdown <frontend/owl/dropdown>`
      - full-featured dropdown
    * - :ref:`Pager <frontend/pager>`
      - a small component to handle pagination
 
+.. _frontend/owl/actionswiper:
+
+ActionSwiper
+------------
+
+.. note:: This component is a mobile feature, only supported in the Enterprise version of Odoo.
+
+Location
+~~~~~~~~
+
+`@web_enterprise/core/action_swiper/action_swiper`
+
+Description
+~~~~~~~~~~~
+
+This is a component that can perform actions when an element is swiped
+horizontally. The swiper is wrapping a target element to add actions to it. 
+The action is executed once the user has released the swiper passed
+half of its width.
+
+.. code-block:: xml
+
+  <ActionSwiper onLeftSwipe="Object" onRightSwipe="Object">
+    <SomeElement/>
+  </ActionSwiper>
+
+The simplest way to use the component is to use it around your target element directly
+in an xml template as shown above. But sometimes, you may want to extend an existing element
+and would not want to duplicate the template. It is possible to do just that.
+
+If you want to extend the behavior of an existing element, you must place the element
+inside, by wrapping it directly. Also, you can conditionnally add props to manage when the
+element might be swipable or not. 
+
+You can use the component to interact easily with records, messages, items in lists and much more.
+
+.. image:: ./images/actionswiper.png
```

Comment:
This image could eventually be compressed a bit with `pngquant` but, given its size, it's not really worth it.